### PR TITLE
Update guide.md for iPad simple setup

### PR DESCRIPTION
**Pull request by AckerApple**
I found it so simple and beautiful to use a different method to get VS Code on my iPad running

### CODE REVIEW

**Review by Mel-5569 — APPROVED** [review]

File: doc/guide.md [hunk 1]
```diff
@@ -115,6 +116,12 @@ There are several approaches to securely operating and exposing `code-server`.
 
 **tip**: You can list the full set of `code-server` options with `code-server --help`
 
+### iPad Simple Setup
+
+Now that you have an exposed code-server instance with an IP and port number, you are one simple step away.
```
Comment (nhooyr) [inline doc/guide.md:121]: What's the point of this app vs just visiting code-server with safari?

Comment (AckerApple) [reply doc/guide.md:121]: So you know how the documentation says iPads don’t work easy and you need a cloud or self signed ssl injection for iPads to support copy/paste and other commands....

Ya, you don’t need any of that with Serveditor.

With this app you don’t have to cloud setup or config a secure certificate setup for a full featured Visual Studio.

This has everything to do with iOS and its https requirements of a true ssl

Comment (nhooyr) [reply doc/guide.md:121]: Wouldn't it just be over `http` then at that point? Or does it access code-server over `ssh`?

I've got self signed certificates working on iPad now! See #2255

Will be in next release.

Comment (AckerApple) [reply doc/guide.md:121]: Whatever protocol it does use, it just works including copy paste and additional features that code server does NOT support unless on https

File: doc/guide.md [hunk 2]
```diff
@@ -115,6 +116,12 @@ There are several approaches to securely operating and exposing `code-server`.
 
 **tip**: You can list the full set of `code-server` options with `code-server --help`
 
+### iPad Simple Setup
```
Comment (nhooyr) [inline doc/guide.md:119]: Let's add a note re this app in `doc/ipad.md` and then I think we can merge!

Comment (AckerApple) [reply doc/guide.md:119]: Ok, I see your callout to edit this file:

https://github.com/cdr/code-server/blob/v3.7.1/doc/ipad.md

I will do just that

**Comment by nhooyr** [comment]
Thank you @AckerApple!

**Comment by AckerApple** [comment]
Thank you so very much. I’ve been so meaning to get back to this. I program every day on code server on my iPad. I can help anyone with more questions

**Pull request merged**